Source/WPEFramework/PluginServer.h

```diff
@@ -661,7 +661,7 @@ namespace PluginHost {
                     Unlock();
 
                     result = Core::proxy_cast<Core::JSONRPC::Message>(Factories::Instance().JSONRPC());
-                    result->Error.SetError(Core::ERROR_BAD_REQUEST);
+                    result->Error.SetError(Core::ERROR_PLUGIN_NOTACTIVATED);
```

Comment:
I think another ERROR here should be ERROR_UNAVAILABLE, as indicated, the plugin might be active but just has no JSONRPC interface.

Reply:
sounds good. We can have ERROR_UNAVAILABLE.